src/winter_onboarding_2021/fleet_management_service/db/fleet.clj

```diff
@@ -1,5 +1,40 @@
 (ns winter-onboarding-2021.fleet-management-service.db.fleet
-  (:require [winter-onboarding-2021.fleet-management-service.db.core :as db]))
+  (:require [winter-onboarding-2021.fleet-management-service.db.core :as db]
+            [honey.sql :as sql]
+            [honey.sql.helpers :refer [select from limit offset join order-by where]]))
 
 (defn create [fleet]
   (db/insert! :fleets fleet))
+
+(defn total []
+  (db/query! (sql/format (-> (select [:%count.* :count])
+                             (from :fleets)))))
+
+(defn select-fleets-for-admin [admin-id off lim]
```

Comment:
`user-fleets` ?

Reply:
I named it `select-fleets-for-admin` because we can only list fleets for an admin. For a manager or a driver, listing fleets is not authorised.

Reply:
@neenaoffline